quetz/tests/test_config.py

```diff
@@ -35,26 +35,29 @@ def test_config_without_file_path_set(config_str):
 
 
 def test_config_users(config):
-    assert config.users_default_role == "member"
-    assert config.users_create_default_channel
-    assert config.users_admins == ["bartosz"]
-    assert not config.users_maintainers
-    assert not config.users_members
+    config = Config()
+    assert config.users.default_role == "member"
+    assert config.users.create_default_channel
+    assert config.users.admins == ["bartosz"]
+    assert not config.users.maintainers
+    assert not config.users.members
 
 
 @pytest.mark.parametrize(
     "config_extra", ["[users]\nadmins=[]", "[users]\nmaintainers=[]"]
 )
 def test_config_empty_users_section(dao: Dao, user, config):
+    config = Config()
     assert config.configured_section("users")
-    assert not config.users_admins
-    assert not config.users_maintainers
-    assert not config.users_members
-    assert not config.users_default_role
-    assert not config.users_create_default_channel
+    assert not config.users.admins
+    assert not config.users.maintainers
+    assert not config.users.members
+    assert not config.users.default_role
+    assert not config.users.create_default_channel
```

Comment:
Accessing fields through the `_` operator will not be possible anymore.

Reply:
I'd say good riddance!